ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/MeteredLedgerDao.scala

```diff
@@ -163,25 +152,54 @@ class MeteredLedgerDao(ledgerDao: LedgerDao, metrics: MetricRegistry)
     val storePartyEntry: Timer = metrics.timer(prefix :+ "store_party_entry")
     val storeInitialState: Timer = metrics.timer(prefix :+ "store_initial_state")
     val storePackageEntry: Timer = metrics.timer(prefix :+ "store_package_entry")
-    val storeLedgerEntry: Timer = metrics.timer(prefix :+ "store_ledger_entry")
+    val storeTransaction: Timer = metrics.timer(prefix :+ "store_ledger_entry")
```

Comment:
Wanna rename this one?

```suggestion
    val storeTransaction: Timer = metrics.timer(prefix :+ "store_transaction")
```

Reply:
This is to retain the current metric name, to not cause disturbance in metric dashboards.

Reply:
Eh, fair enough.

Reply:
I did it on purpose, I believe @mziolekda wants them to be stable until we have a plan to change them all at once with detailed documentation.

Reply:
Well, hopefully the only disturbance will be the smaller numbers from the same metric. :stuck_out_tongue: